### Make a move using Standard Algebraic Notation.
#
# This service can also be used by a robot node (sub)system to initiate a
# physical chess move.
#
# After making a valid chess move, the engine will automatically advance
# to the next player.
#
# The requesting player's color must match the current color's play.
#
# This service can be used to determine whether a move is valid by looking at
# the result field in the response move message. A valid move advances the play,
# however. To undo, an UndoMove request must be made to back up one ply.
#
# Note that the response time is dependent of the difficulty level of the game
# and can be very long if the search depth is deep.
#

ChessColor  player # requesting player: white or black
string      SAN    # string of format: {file}{rank}{file}{rank}. Example: "a2a4"
---
ChessMove move     # engine's fully quantified move
